### ISSUE TITLE
Put user on org when they login via SAML

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#9375]
When a user logs in via SAML they should be put on the organization that the SAML record is attached to.
We can grab the orgId from the SAML table.
We can ensure org membership every time a user logs in via SAML, or we could do it just when they sign up. I think we should do the former because as long as they are a part of the company's SAML, they should be a part of the parabol SAML.

AC
- [ ] When a user logs in via SAML they should be put on the organization that the SAML record is attached to.

**Comment by Dschoordsch** [comment]
For now we should limit this to orgs that also have the `publicTeams` feature flag so users are not lost and have an easy way forward.

**Comment by mattkrick** [comment]
yeah, i like that. i'll just postpone until publicTeams goes GA

**Comment by jordanh** [comment]
Dupicate of #9411 ?

**Comment by Dschoordsch** [comment]
No. Prompt to join org will send a notification and ask a new user if they want to join an organization which matches their domain. This issue is about not asking but just putting athe user in the existing org directly on sign up.

**Comment by jordanh** [comment]
Groomed. I am going to keep this one open as well.
